app/scripts/background.js

```diff
@@ -208,6 +247,18 @@ function setupController (initState, initLangCode) {
     return popupIsOpen || Boolean(Object.keys(openMetamaskTabsIDs).length) || notificationIsOpen
   }
 
+  /*
+   * A runtime.Port object, as provided by the browser:
+   * https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/runtime/Port
```

Comment:
You can use the `@link` tag to handle reference URLs for special formatting with some parsers.